### DISCUSSION TITLE
How do you change camera position?

### DISCUSSION THREAD

**Discussion by AlmostMikhail** [friction2d/friction#376] [Q&A]
I want to change from 1080x1920 to 1920x1080 but messes relative position of objects to camera

**Comment by pgilfernandez** [comment]
I would put all project assets into a group so that, after you change the canvas dimensions, you can grab the group and place it wherever you want.
Finally, you can "ungroup (keep transform)" to get rid of the group.

I hope it helps you... if not, share the project with us and explain it briefly what do you want to accomplish.

Cheers

**Reply by AlmostMikhail** [reply]
Thanks
